panacea/src/pml_parser.yrl

```diff
@@ -169,19 +169,33 @@ operator -> '>=' : '$1'.
 
 Erlang code.
 
-extract_string({_,_,Str}) ->
+extract_string({_, _, Str}) ->
     strip_quotes(Str).
 
 strip_quotes(Str) ->
     CharList = string:strip(Str, both, $"),
     list_to_binary(CharList).
 
-construct({Type, Line}, Attributes, Value) ->
+number_expression({_, Line, Num}) ->
```

Comment:
I don't think we need to bother parsing numbers because we never actually use them.